resources/api/openapi.yml

```diff
@@ -556,6 +595,61 @@ paths:
         '404':
           $ref: '#/components/responses/NotFoundError'
 
+  /shifttypes:
```

Comment:
<img width="580" alt="Screenshot 2024-06-09 at 18 47 38" src="https://github.com/engelsystem/engelsystem/assets/73076134/20ae1c7d-74fe-4d1f-8a2c-3fcb87563b4b">

I'm not sure if the alphabetic order is really usefull, I would prefer an order by id

Reply:
I don't